apps/chat/src/components/AppsEditor/Settings/Previews/ApplicationPreviewChat.tsx

```diff
@@ -18,6 +18,8 @@ import { Spinner } from '@/src/components/Common/Spinner';
 import { UploadStatus } from '@epam/ai-dial-shared';
 
 interface Props {
+  handlePreviewMouseLeave: () => void;
+  handlePreviewMouseEnter: () => void;
```

Comment:
```suggestion
  onPreviewMouseLeave: () => void;
  onPreviewMouseEnter: () => void;
```

lets follow our function props convention